zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+By default, a Governance Proposal requires a three-fifths Approval Threshold, and MUST have a Deadline at least two weeks from when the Proposal is published on-chain.
+
+All Consituencies MUST be consulted in order to determine whether they consider a draft Proposal to be "controversial", before it published on-chain. If *any* Constituency decides that a Proposal should be treated as controversial, it is then subject to a higher Approval Threshold of three quarters, and a Deadline at least four weeks from when the Proposal is published on-chain. (In the case of the Coin-holder constituency, a coin-holder vote is not necessarily required to establish this; it is likely sufficient to decide based on forum discussions for example.)
+
+The following categories of Governance Proposal are automatically subject to the three-quarters Approval Threshold and 4-week Deadline:
+
+* Increasing or decreasing the voting power of zBloc Constituencies by adjusting how many Voting Units they hold.
+* Increasing or decreasing the number of Funding Pools, the rules that govern them, and the amounts allocated.
+
+In the case of Governance Proposals that affect consensus, final ratification occurs when node operators adopt the software implementing the changes.
+
+Note: Nothing forces developers of Zcash consensus node software to follow this specification. The aim of a process specification like this one is only to establish a social consensus. It fundamentally cannot affect the autonomy of developers of Zcash consensus node software to publish (or not) the software they want to publish, or the autonomy of node operators to run (or not) the software they want to run.
+
+## Proposals
+
+As stated in [Terminology], there are two categories of Proposal that can be put to a vote of the [zBloc Constituencies](#zbloc-membership):
+
+* a "Governance Proposal", expressing a proposed change in Zcash Governance;
+* a "Funding Proposal", expressing a proposed disbursement of Protocol-defined Ecosystem Funding.
+
+Proposals have a Deadline (expressed as a Deadline Height if [^draft-ecc-onchain-accountable-voting] is used). If the Deadline is reached, they are implicitly Rejected. This does not necessarily preclude reintroducing the same or a similar Proposal.
+
+### Governance Proposals
+
+At least the following changes to Zcash Governance MUST be proposed and approved by a Governance Proposal before they can take effect:
+
+* Changes to Zcash consensus rules. In this case Approval is REQUIRED in order for an activation height to be set.
+* Substantive changes to the ZIP Process defined in ZIP 0 [^zip-0000].
+* Substantive changes to this ZIP defining the zBloc.
+* Changes to the set of zBloc Constituencies and/or the distribution of Voting Units among those Constituencies.
+* Changes to the policy a Constituency uses for its decision-making.
```

Comment:
This last one should only apply to the coinholder constituency; it should not require a Governance Proposal for ECC to change its internal decision-making mechanism.

Reply:
Oh, I misinterpreted Josh's intent here then (given his thumbs-up).